src/sentry/static/sentry/app/sentryTypes.jsx

```diff
@@ -93,6 +93,31 @@ export const DiscoverSavedQuery = PropTypes.shape({
   ...DiscoverQueryShape,
 });
 
+const DiscoverResultShape = {
+  data: PropTypes.arrayOf(PropTypes.object),
+  meta: PropTypes.arrayOf(
+    PropTypes.shape({
+      type: PropTypes.string,
+      name: PropTypes.string,
+    })
+  ),
+  timing: PropTypes.shape({
+    duration_ms: PropTypes.number,
+    marks_ms: PropTypes.shape({
```

Comment:
Should we just do `PropTypes.object` here for now - I didn't realise this was being returned until now and it might be better to just exclude it from the api response